docs/src/test-api/class-testconfig.md

```diff
@@ -36,6 +36,26 @@ export default defineConfig({
 });
 ```
 
+## property: TestConfig.captureGitInfo
+* since: v1.51
+- type: ?<[Object]>
+  - `commit` ?<boolean> Whether to capture commit information such as hash, author, timestamp.
+  - `diff` ?<boolean> Whether to capture commit diff.
+
+* These settings control whether git information is captured and stored in the config [`property: TestConfig.metadata`].
+* The structure of the git commit metadata is not documented and is a subject to change.
+* Default values for these settings depend on the environment. When tests run as a part of CI where it is safe to obtain git information, the default value is true, false otherise.
```

Comment:
```suggestion
* Default values for these settings depend on the environment. When tests run as a part of CI where it is safe to obtain git information, the default value is true, false otherwise.
```